<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>otherPlayer</class>
 <widget class="QWidget" name="otherPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>otherPlayer</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>1000</width>
     <height>411</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#scrollPoks{background-image: url(:/pokpic/pokemon.png);
background-color: rgb(133,198,98);}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollPoks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>399</height>
     </rect>
    </property>
    <widget class="QLabel" name="pokLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华光胖头鱼_CNKI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TA的精灵：</string>
     </property>
    </widget>
    <widget class="QLabel" name="winRate">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华光胖头鱼_CNKI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pokemonClientGUI.qrc"/>
 </resources>
 <connections/>
</ui>
